modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -87,8 +86,11 @@ protected Integer getLivenessCheckPort() {
     @Override
     protected void configure() {
 
-        checkState(desiredCapabilities != null);
-        if (! customImageNameIsSet) {
+        if(desiredCapabilities == null) {
+            throw new IllegalStateException();
```

Comment:
see #390 where this change was introduced (the whole branch is based on #390, see the description)